step-framework-server/src/main/java/step/framework/server/tables/Table.java

```diff
@@ -18,6 +19,7 @@ public class Table<T> {
 
     private final boolean filtered;
     private BiFunction<TableParameters, Session<?>, Filter> tableFiltersFactory;
+    private Function<ArrayList<Filter>, Filter> derivedTableFiltersFactory;
```

Comment:
Why specifically an ArrayList? It's best practice to just use the base type (List) unless there's a good reason to require a specific implementation.

Reply:
Because it's a copy/paste, I'll change that.